tox.ini

```diff
@@ -50,3 +51,4 @@ commands =
     coverage: -coverage combine
     coverage: coverage report
     template: python3 bin/make_template {posargs}
+    dockercompose: docker-compose {posargs}
```

Comment:
The `docker-compose` command is currently only added to `tox.ini` for apps. It needs to be added for any project (but only if `"services"` is `"yes"`)